include/rdma/fabric.h

```diff
@@ -451,6 +454,7 @@ enum fi_type {
 	FI_TYPE_ADDR_FORMAT,
 	FI_TYPE_TX_ATTR,
 	FI_TYPE_RX_ATTR,
+	FI_TYPE_SRX_ATTR,
```

Comment:
All of the changes to this file break the ABI.  Please drop this patch as it breaks all existing applications.